### PR TITLE
issue #299: Handle server frames also when ReadyState is SR_Closing 

### PR DESCRIPTION
**Pull request by jakobsa**
Instead of returning the handlers if ReadyState is SR_Closing returns only on ReadyState SR_Closed

Now the close handshake succeeds.

### CODE REVIEW

File: SocketRocket/SRWebSocket.m [hunk 1]
```diff
@@ -1649,7 +1649,7 @@ - (NSString *)SR_origin;
 
 @end
 
-//#define SR_ENABLE_LOG
+#define SR_ENABLE_LOG
```
Comment (dfed) [inline SocketRocket/SRWebSocket.m]: This change should likely not be included.

**Comment by dfed** [comment]
That's for the PR! This looks like a reasonable change.

To get this merged, please do the following:
1) Sign our [CLA](https://spreadsheets.google.com/spreadsheet/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1)
2) Comment out `#define SR_ENABLE_LOG`

**Comment by jakobsa** [comment]
done

**Comment by dfed** [comment]
Thanks! Will merge this Friday when I roll a new cocoa pod version.

**Comment by jakobsa** [comment]
any update on this?

**Comment by dfed** [comment]
Thanks for the ping! Handling this now.

**Comment by dfed** [comment]
Looks like 0.4.2 was already claimed. This will make it into 0.4.3

**Pull request merged**